docker-compose.yml

```diff
@@ -23,4 +25,5 @@ services:
       - OCTOBOX_DATABASE_USER=postgres
       - OCTOBOX_DATABASE_PASSWORD=development
       - OCTOBOX_DATABASE_HOST=database.service.octobox.internal
+      - MEMCACHEDCLOUD_SERVERS=memcached.service.octobox.internal
```

Comment:
Agreed, would be good to make the docker setup usable for dev, test and production